07_git_exercises/shlomigd_ex1/README

```diff
@@ -0,0 +1,68 @@
+Git Basics:
+
+1. cat > abc.txt
+   1
+   ctrl d
+
+2.Red.
+3.Green.
+    git status
+    git add abc.txt
+    git status
+    git commit abc.txt -m "file created"
```

Comment:
No need to specify the filename abc.txt in the commit (it is stages already in the `git add` command)